community/cloud-foundation/templates/cloudbuild/trigger.py.schema

```diff
@@ -0,0 +1,429 @@
+                dir, which specifies an absolute path, the RepoSource dir is 
+                ignored for the step's execution.
+            id:
+              type: string
+              description: |
+                The unique identifier for the build step. Used in waitFor to
+                reference this build step as a dependency.
+            waitFor:
+              type: array
+              description: |
+                The ID(s) of the step(s) this build step depends on.
+                The build step will not start until all the build steps in 
+                waitFor have completed successfully. If waitFor is empty, 
+                this build step will start when all the previous build steps 
+                in the Build.Steps list have completed successfully.
+              items:
+                type: string
+            entrypoint:
+              type: string
+              description: |
+                The entry point to be used instead of the build step image's 
+                default entry point. If not set, the image's default entry 
+                point is used.
+            secretEnv:
+              type: array
+              description: |
+                The list of environment variables are encrypted using the 
+                Cloud Key Management Service crypto key. These values must be 
+                specified in the build's Secret.
+              items:
+                type: string
+            volumes:
+              type: array
+              description: The list of volumes to mount into the build step.
+              items:
+                type: object
+                properties:
+                  name:
+                    type: string
+                    description: |
+                      The name of the volume to mount. Volume names must be 
+                      unique per build step, and must be valid names for Docker
+                      volumes. Each named volume must be used by at least two
+                      build steps.
+                  path:
+                    type: string
+                    description: |
+                      The path to mount the volume at. Paths must be absolute
+                      and cannot conflict with other volume paths on the same 
+                      build step or with certain reserved volume paths.
+            timeout:
+              type: string
+              description: |
+                The time limit for executing the build step. If not defined,
+                the step has no time limit, and is allowed to continue to run
+                until it either completes or the build itself times out.
+                Defined in seconds with up to nine fractional digits,
+                terminated by 's'; for example: 3.5s.
+      timeout:
+        type: string
+        description: |
+          The time limit for executing the build step. If not defined, the
```

Comment:
Is this overall timeout for entire build? I think this description is incorrect - it's outside of the steps object (the one above it is inside steps object).

Reply:
Fix in commit `b806511`